### DISCUSSION TITLE
Release Arm64 binaries for the MSVC Windows flavor

### DISCUSSION THREAD

**Discussion by prayaas-a** [jedisct1/libsodium#1219] [General]
Hi. I see that as of #1130, libsodium has supported Arm64 builds with MSVC. It should work with both VS 2019 and VS 2022 (though the latter has missing project configurations).

Are there any plans to release Arm64 binaries for libsodium? If you're not planning on releasing a 1.0.19 sometime soon, perhaps there could be a one-off re-release of 1.0.18 (going back to that release's tag, but also built for Arm64 for the MSVC version).

There are several projects like [pyca/pynacl](https://github.com/pyca/pynacl/issues/775) that depend on libsodium Arm64 binaries being available to be able to release Arm64 binaries for their project. 🙂 Thanks.

**Comment by jedisct1** [comment]
Unfortunately, these changes didn't work. At least they never passed CI, and I never had a way to check that libsodium actually worked on Windows/aarch64.

Do you know of a CI system that supports Windows/aarch64?

Also, do the libraries [created using zig](https://doc.libsodium.org/installation#compiling-with-zig) work as a drop-in replacement for the ones created using Visual Studio? If that is the case, we could avoid Visual Studio, and shipping *totally untested* pre-compiled versions for aarch64 would be easy.
